modules/social_features/social_group/modules/social_group_default_route/social_group_default_route.services.yml

```diff
@@ -1,10 +1,13 @@
 services:
   social_group_default_route.redirect_subscriber:
-    class: Drupal\social_group_default_route\EventSubscriber\RedirectSubscriber
```

Comment:
@rochek03 As I understand you want to have common prefix `SocialGroupDefaultRoute`, but since we already have namespace and in most of other places we already use just `RedirectSubscriber` I think it's better to not change it if there is no reason for it. What do you think?

Reply:
@ribel Actually, this class isn't used anywhere, but I think you are right - to change the class name it isn't a good practice, so I reverted that class name